packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -26,6 +27,11 @@ import {
 	IProvideFluidDataStoreRegistry,
 } from "@fluidframework/runtime-definitions";
 
+export interface IContainerRuntimeWithResolveHandle_Deprecated extends IContainerRuntime {
```

Comment:
@markfields Should the GC doc reference this interface or the `ContainerRuntime` class for https://github.com/microsoft/FluidFramework/pull/17171#discussion_r1317709194

Reply:
Probably this interface